src/store/auth/sagas.ts

```diff
@@ -102,15 +102,61 @@ function * loginUserWorker () {
     const user = profile.user;
     const userId = user.id;
     const userName = user.name.split(" ");
-    const currentOrg = profile.current_org;
+
+    let currentOrg = {
+      id: "",
+      name: "",
+      role: {
+        id: "",
+        name: ROLES.baseUser.value,
+      },
+    };
+
+    const currentUser: string | null = localStorage.getItem("currentUser");
+
+    if (currentUser && currentUser !== userId) {
+      localStorage.removeItem("orgInStorage");
+      
+      localStorage.setItem("currentUser", userId);
+    }
+
+    const orgInStorage: string | null = localStorage.getItem("orgInStorage");
```

Comment:
As mentioned in the other comment move the localStorage to a function in util to be used here.